src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,137 @@
+package duke;//package duke;
+//
+///**
+// * Duke is a chatbot that takes in tasks, which includes todos, deadlines and events, and compiles them
+// * in a list. Tasks can be removed from the list and be marked as completed.
```

Comment:
Just a friendly reminder that there should not be large chunks of commented code present as mentioned in the iP grading rubrics. You should delete them if not needed or transfer them somewhere else 😄